pkgs/applications/misc/1password-gui/default.nix

```diff
@@ -12,7 +12,8 @@ let
   pname = "1password";
 
   versions = builtins.fromJSON (builtins.readFile ./versions.json);
-  inherit (versions.${channel} or (throw "unknown channel ${channel}")) version;
+  hostOs = if stdenv.hostPlatform.isLinux then "linux" else "darwin";
```

Comment:
Uhm. Should I use `targetPlatform` here instead?

Reply:
I think `hostPlaform` is accurate, it's what we're trying to build.